tests/Integration/Behaviour/Features/Scenario/Order/duplicate_products.feature

```diff
@@ -1,4 +1,4 @@
-# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s order --tags duplicate-products-in-order
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags duplicate-products-in-order
```

Comment:
are you sure? I think it was intentional to stay in order. Its under Order scenario and its about duplicating a product inisde the order (its only related to orders)
```suggestion
# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s order --tags duplicate-products-in-order
```

Reply:
Oh right this feature is actually not related to the product domain but to the order, nice catch!

Reply:
@jolelievre ask me to change it. Personally I absolutely don't know which one is the good one!

Reply:
This one is part of the order suite